Sources/KlaviyoSDKDependencies/CasePaths/CasePathable.swift

```diff
@@ -0,0 +1,565 @@
+import KlaviyoSDKDependencies
+
```

Comment:
```suggestion
```
we shouldn't need this because it's already part of the `KlaviyoSDKDependencies` package